test/hosting/test_HostingBasic.cs

```diff
@@ -51,5 +54,89 @@ public static void TestCommandFromCore()
                 }
             }
         }
+
+        [SkippableFact]
+        public static void TestCommandFromMMI()
+        {
+            // Test is disabled since we do not have a CimCmdlets module released in the SDK.
+            Skip.IfNot(Platform.IsWindows);
+            using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
+            {
+                var results = ps.AddScript("[Microsoft.Management.Infrastructure.CimInstance]::new('Win32_Process')").Invoke();
+                Assert.True(results.Count > 0);
+            }
+        }
+
+        [SkippableFact]
+        public static void TestCommandFromDiagnostics()
+        {
+            Skip.IfNot(Platform.IsWindows);
+            using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
+            {
+                var results = ps.AddScript("Get-WinEvent -ListLog Application").Invoke();
+
+                foreach (dynamic item in results)
+                {
+                    Assert.Equal("Application", item.LogName);
+                }
+            }
+        }
+
+        [SkippableFact]
+        public static void TestCommandFromSecurity()
+        {
+            Skip.IfNot(Platform.IsWindows);
+            using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
+            {
+                var results = ps.AddScript("ConvertTo-SecureString -String test -AsPlainText -Force").Invoke<SecureString>();
+                Assert.IsType<SecureString>(results[0]);
+            }
+        }
+
+        [SkippableFact]
+        public static void TestCommandFromWSMan()
+        {
+            Skip.IfNot(Platform.IsWindows);
+            using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
+            {
+                var results = ps.AddScript("Test-WSMan").Invoke();
+
+                foreach (dynamic item in results)
+                {
+                    Assert.Equal("Microsoft Corporation", item.ProductVendor);
+                }
+            }
+        }
+
+        [Fact]
+        public static void TestCommandFromNative()
+        {
+            var fs = File.Create(Path.GetTempFileName());
+            fs.Close();
+
+            string target = fs.Name;
+            string path = Path.GetTempFileName();
+
+            using (System.Management.Automation.PowerShell ps = System.Management.Automation.PowerShell.Create())
+            {
+                string command = $"New-Item -ItemType SymbolicLink -Path {path} -Target {target}";
```

Comment:
Is this test because symlink depends on libpsl?  Perhaps add a comment as `Native` may be interpreted as a native command not from `Microsoft.PowerShell.Native`

Reply:
I will add a comment.